src/coreclr/tools/Common/TypeSystem/Common/TargetDetails.cs

```diff
@@ -9,37 +9,41 @@ namespace Internal.TypeSystem
     /// <summary>
     /// Specifies the target ABI.
     /// </summary>
-    public enum TargetOS
+    /// This is used in serialization for symbols. Please don't reorder or reassing values.
+    /// Deletions and renames are OK, additions are only allowed as appends.
+    public enum TargetOS : uint
     {
-        Unknown,
-        Windows,
-        Linux,
-        OSX,
-        FreeBSD,
-        NetBSD,
-        SunOS,
-        WebAssembly,
+        Unknown     = 0,
```

Comment:
Enums that are part of the TypeSystem are not considered to be part of the File format, and should not be considered to be stable. Please make separate enums in a file which is explicitly for the file format you are working with.